### ISSUE TITLE
@@map and @map not mixing

### ISSUE THREAD

**Issue by afroborg** [jamiepine/prismix#18]
I have the following structure for a one-to-many relation between `users` and `articles`.

**user.prisma**:

```
model User {
	username String @id
	firstName String @map("first_name")
	lastName String @map("last_name")

	@@map("users")
}
```

**article.prisma**:

```
model Article {
	id Int @id @default(autoincrement())
	title String
	body String

	author User @relation(fields: [refAuthor], references: [username])
	refAuthor String @map("ref_author")

	@@map("articles")
}

model User {
	username String @id
	articles Article[]
}
```

However, when I try to mix the schemas the `@@map` and `@map` for **user.prisma** is not set?

**schema.prisma**:

```
model User {
  username          String             @id
  articles          Article[]          @relation(name: "ArticleToUser")
  firstName         String
  lastName          String
}
```

**Comment by ddhorstman** [comment]
+1 to this. I'm very excited about `prismix`, but I need this bug to be fixed before I can adopt it.

**Comment by jamiepine** [comment]
This is now fixed in 1.0.21 thanks to @ddhorstman
